manifests/m/MullvadVPN/MullvadVPN/2025.6/MullvadVPN.MullvadVPN.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with komac v2.11.2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: MullvadVPN.MullvadVPN
+PackageVersion: '2025.6'
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ProductCode: 2A356FD4-03B7-4F45-99B4-737BE580DC82
+ReleaseDate: 2025-05-19
+AppsAndFeaturesEntries:
+- DisplayName: Mullvad VPN 2025.6.0
+  ProductCode: 2A356FD4-03B7-4F45-99B4-737BE580DC82

```

Comment:
I don't think this entire entry is needed, it should be intuited on its own. In any case, the duplicate product code should definitely be avoided.